PIM/Operation/02_ManageVariants.md

```diff
@@ -0,0 +1,215 @@
+
+#### Prerequisites
+
+A variant set has been created to the attribute set of the product you want to assign the variants to, see [Create a variant set](../Integration/07_ManageVariantSets.md#create-a-variant-set).
+
+#### Procedure
+*PIM > Products > Tab LIST > Button Add > Tab Variants > Select variant set > Button Add*   
+*PIM > Products > Tab LIST > Select Product > Tab Variants > Button Add*
+
+![Add variants](../../Assets/Screenshots/PIM/Products/List/Variants/AddVariantCreate.png "[Add variants]")
+
+1. Click the [Add multiple variants ![Add multiple variants](../../Assets/Icons/Documents.png "[Add multiple variants]") ] button.   
+  The *Select Values* wizard window is displayed.
+
+  ![Select values wizard](../../Assets/Screenshots/PIM/Products/List/Variants/AddMultipleVariants01.png "[Select values wizard]")
+
+2. Select a value for the displayed attributes in the box(es). The fields and the box(es) differ depending on the selection of defining attributes for the variant set.   
+  The selected attribute value is displayed in the box(es).
+
+3. Click the [ADD VALUE] button in the upper right corner of the box to add a further variant with another value.   
+  A further row is displayed in the box.
+
+  > [Info] If you want to add a variant for each existing value, click the [ADD ALL VALUES] button in the box. This button is only available for certain attributes.
+
+4. Select a value for the attribute in the new row.
+  The selected attribute value is displayed in the box.
+
+  > [Info] Repeat the steps **3** and **4** to add further variants.
+
+5. Click the [NEXT] button in the bottom right corner of the window.   
+  The *Summary* wizard window is displayed.
+
+  ![Summary wizard](../../Assets/Screenshots/PIM/Products/List/Variants/AddMultipleVariants02.png "[Summary wizard]")
+
+6. Check if all variants listed in the *Variants that will be created (ignoring duplicated)* box are correct.
+
+  > [Info] To remove a variant from the list, select the checkbox of the variant and click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button in the editing toolbar displayed above the list.
+
+7. Click the [FINISH] button in the bottom right corner.   
+  The new variants have been added to the product. The *Summary* wizard window is closed. The variants are displayed in the variants list.
+
+  ![Variants added](../../Assets/Screenshots/PIM/Products/List/Variants/VariantsAddedCreate.png "[Variants added]")
+
+8. Click the [SAVE] button in the upper right corner.   
+  The variants have been saved. The product view is closed. The variants are displayed in the product list when you enable the **All products** option in the *Variants* drop-down list at the top of the product list.
+
+  > [Info] For detailed information about the different views, see [Manage the view](./04_ManageView.md).
+
+
+
+## Add variants
+
+You can add a single or multiple variants to a product directly when you create the master product.
```

Comment:
```suggestion
You can add a single or multiple variants to a master product after it has been created/after creating a master product.
```
Stimmt es so? 
Vgl. ## Create variants Einführungssatz -> You can create a single or multiple variants to a product directly when you create the master product. vs ## Add variants Prerequisites -> At least one product has been created.